pkg/logs/status/status.go

```diff
@@ -61,6 +61,7 @@ func Get() Status {
 	}
 	// Convert to json
 	var integrations []Integration
+	warningsDeduplicator := make(map[string]bool)
```

Comment:
If a warning/message was not tight to a source, you'd need to do any deduplicate here.

Reply:
Yes, but the alternative would have been to compute the warning at the status create but this would require computer to identify LogSource for file an count the inputs 

I found it messier than just deduplicating warning messages (which is quite a small operation)